.gitignore

```diff
@@ -5,3 +5,6 @@ dist
 bower_components
 *~
 *.swp
+
+.idea/
```

Comment:
This should be handled in your [global gitignore](https://help.github.com/articles/ignoring-files/).